cmf-cli/resources/template_feed/iot/from1000/IoT.Package/%iotpackages%/cmfpackage.json

```diff
@@ -7,7 +7,7 @@
   "isUniqueInstall": false,
   "contentToPack": [
     {
-      "source": "dist/*",
```

Comment:
aren't you going to pack also the settings typescript files if you do this?

Reply:
For V10 we only need the runtime component as the GUI is now addressed in compile time. It's true that we are adding a bit of trash to the package. Maybe we can discuss this, to see if there are alternatives